# Hekateros hekateros_control/GotoZeroPt service

---
int8 rc   # return code
